TeachingRecordSystem/src/TeachingRecordSystem.Core/Jobs/EwcWalesImport/InductionImporter.cs

```diff
@@ -0,0 +1,452 @@
+            OrganisationMatchStatus = orgMatchStatus,
+            OrganisationId = organisationId
+        };
+        return lookupData;
+    }
+
+    public (List<string> ValidationFailures, List<string> Errors) Validate(EwcWalesInductionImportData row, InductionImportLookupData lookups)
+    {
+        var validationFailures = new List<string>();
+        var errors = new List<string>();
+
+        //ReferenceNumber/Trn
+        if (String.IsNullOrEmpty(row.ReferenceNumber))
+        {
+            errors.Add("Missing Reference No");
+        }
+
+        //Date Of birth
+        if (String.IsNullOrEmpty(row.DateOfBirth))
+        {
+            errors.Add("Missing Date of Birth");
+        }
+        else
+        {
+            if (!DateOnly.TryParseExact(row.DateOfBirth, "dd/MM/yyyy", CultureInfo.InvariantCulture, DateTimeStyles.None, out _))
+            {
+                errors.Add("Validation Failed: Invalid Date of Birth");
+            }
+        }
+
+        //InductionPassedDate
+        if (String.IsNullOrEmpty(row.StartDate))
+        {
+            errors.Add("Missing Induction Start date");
+        }
+        else
+        {
+            if (!DateOnly.TryParseExact(row.StartDate, "dd/MM/yyyy", CultureInfo.InvariantCulture, DateTimeStyles.None, out _))
+            {
+                errors.Add("Validation Failed: Invalid Induction start date");
+            }
+        }
+
+        //InductionPassedDate
+        if (String.IsNullOrEmpty(row.PassedDate))
+        {
+            errors.Add("Missing Induction passed date");
+        }
+        else
+        {
+            if (!DateOnly.TryParseExact(row.PassedDate, "dd/MM/yyyy", CultureInfo.InvariantCulture, DateTimeStyles.None, out _))
+            {
+                errors.Add("Validation Failed: Invalid Induction passed date");
+            }
+        }
+
+        switch (lookups.PersonMatchStatus)
+        {
+            case EwcWalesMatchStatus.NoAssociatedQts:
+                break;
+            case EwcWalesMatchStatus.NoMatch:
+                errors.Add($"Teacher with TRN {row.ReferenceNumber} was not found.");
+                break;
+            case EwcWalesMatchStatus.TrnAndDateOfBirthMatchFailed:
+                errors.Add($"For TRN {row.ReferenceNumber} Date of Birth does not match with the existing record."); //No Test
+                break;
+            case EwcWalesMatchStatus.MultipleTrnMatched:
+                errors.Add($"TRN {row.ReferenceNumber} was matched to more than one record in the system.");
+                break;
+            case EwcWalesMatchStatus.TeacherInactive:
+                errors.Add($"Teacher with TRN {row.ReferenceNumber} is inactive."); //No Test
+                break;
+            case EwcWalesMatchStatus.TeacherHasQts:
+                errors.Add($"Teacher with TRN {row.ReferenceNumber} has QTS already.");
+                break;
+        }
+
+        if (!string.IsNullOrEmpty(row.EmployerCode))
```

Comment:
Is `EmployerCode` ever actually used?

Reply:
Yep, that's where account number is stored.

Reply:
> Yep, that's where account number is stored.

Lookup seems to be done on `EmployerName` not `EmployerCode`.